src/api/spec/support/capybara.rb

```diff
@@ -12,6 +12,7 @@
     browser_options = ::Selenium::WebDriver::Chrome::Options.new
     browser_options.args << '--headless'
     browser_options.args << '--no-sandbox' # to run in docker
+    browser_options.args << '--window-size=1280,1024'
```

Comment:
What was it before?

Reply:
I couldn't find that in the documentation, but it looked like ~500px